src/test/java/Homework19.java

```diff
@@ -59,3 +65,34 @@ public static void confirmDeleted() {
 }
 
 
+
+    public void DeletePlaylistTest() throws InterruptedException {
+        String playlistName = "test playlist";
+
+        login("demo@class.com", "te$t$tudent");
+        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//i[@title='Create a new playlist']"))).click();
+
+        WebElement newPlaylist = driver.findElement(By.xpath("//li[text()='New Playlist']"));
+        newPlaylist.click();
+
+        WebElement nameField = driver.findElement(By.xpath("//input[@name='name']"));
+        nameField.clear();
+        nameField.sendKeys(playlistName, Keys.ENTER);
+
+        WebElement testPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li/a[text()='"+playlistName+"']")));
+        testPlaylist.click();
+
+        WebElement deletePlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@title='Delete this playlist']")));
+        deletePlaylist.click();
+
+        Thread.sleep(2000);
+        List<WebElement> playlistNames = driver.findElements(By.xpath("//section[@id='playlists']//li/a"));
+
+        for(WebElement p : playlistNames) {
+            String name = p.getText();
+            if (name.equals(playlistName)) {
+                Assert.assertTrue(false);
+            }
+        }
+    }
+}
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 
![image](https://user-images.githubusercontent.com/104761482/212462711-30a9af52-786e-42ad-b9f4-57e9258909f1.png)

![image](https://user-images.githubusercontent.com/104761482/212462821-b9f70d80-184f-46d7-b5d4-6ed7f9b7b75f.png)